Widgets/widget-TasksByState.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "tasks-state",
+  "version": -1,
+  "modified": "2018-01-14T22:59:42.479686738Z",
+  "name": "Tasks By State",
+  "dataType": "tasks",
+  "widgetType": "column",
+  "query": "replaced:f" ,
```

Comment:
@aviadl is `replaced:f` means only active tasks?

Reply:
Yes